### ISSUE TITLE
esRDD Implicit with all 3 parameters (resource, query, config)

### ISSUE THREAD

**Issue by randallwhitman** [elastic/elasticsearch-hadoop#592]
When calling `sc.esRDD` I need to specify all of resource, query, and config.  I'd like to call the implicit rather than explicitly calling `EsSpark(sc, resource, query, config)`.  In `spark/package.scala` I find:
*   `def esRDD()`
*   `def esRDD(resource: String)`
*   `def esRDD(resource: String, query: String)`
*   `def esRDD(cfg: scala.collection.Map[String, String])`
*   `def esRDD(resource: String, cfg: scala.collection.Map[String, String])`

but not a 3-parameter overload - which can be added as:
```scala
    def esRDD(resource: String, query: String, cfg: scala.collection.Map[String, String]) =
        EsSpark.esRDD(sc, resource, query, cfg)
```
```scala
    def esJsonRDD(resource: String, query: String, cfg: scala.collection.Map[String, String]) =
        EsSpark.esJsonRDD(sc, resource, query, cfg)
```

**Comment by randallwhitman** [comment]
Do I understand correctly that https://github.com/elastic/elasticsearch-hadoop/commit/a97fa9fcc7ddc84478e1200338587d95cf412653 resolves this issue for v2.1.3 and v2.2.x ?

**Comment by costin** [comment]
@randallwhitman That is correct. The [nightly builds](https://www.elastic.co/guide/en/elasticsearch/hadoop/master/install.html#download-dev) have been also published so you can try them out yourself.
